Tests.Unit.Core.ApplicationServices/DomainServices/SystemUsage/ItSystemUsageOverviewReadModelUpdateTest.cs

```diff
@@ -47,7 +47,7 @@ public ItSystemUsageOverviewReadModelUpdateTest()
             _archivePeriodReadModelRepository = new Mock<IGenericRepository<ItSystemUsageOverviewArchivePeriodReadModel>>();
             _dataProcessingReadModelRepository = new Mock<IGenericRepository<ItSystemUsageOverviewDataProcessingRegistrationReadModel>>();
             _interfacesReadModelRepository = new Mock<IGenericRepository<ItSystemUsageOverviewInterfaceReadModel>>();
-            _itSystemUsageReadModelRepository = new Mock<IGenericRepository<ItSystemUsageOverviewItSystemUsageReadModel>>();
+            _itSystemUsageReadModelRepository = new Mock<IGenericRepository<ItSystemUsageOverviewUsedBySystemUsageReadModel>>();
```

Comment:
Er det nok kun med "UsedBy" repository delen her? Skal der ikke også "Using" repository til?

Reply:
Det kommer jo helt an på om der er en test der bruger den (den tilføjes faktisk i ctor nedenfor) ;). Det er der i øvrigt heller ikke for den "anden vej" for repo bruges kun aktivt når der slettes instanser